client/packages/host/src/components/Initialise/hooks.ts

```diff
@@ -61,6 +63,16 @@ export const useInitialiseForm = () => {
     refetchInterval: POLLING_INTERVAL,
     enabled: isPolling,
   });
+  const t = useTranslation('app');
+  const parseErrorMessage = (error: Error, defaultKey: LocaleKey) => {
```

Comment:
this is just a suggestion! I've committed to 7e7f271a3 as there were two files to update and it was getting tricky to summarise in a comment.

Basically, I was going to suggest that we enable translation for the errors, and then got carried away and surfaced the actual errors. uh.. sorry?

Reply:
Looks fine to me!

Thanks Mark! 😺